tests/vmipreset_test.go

```diff
@@ -414,6 +414,81 @@ var _ = Describe("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		})
 	})
 
+	Context("[rfe_id:613]MatchLabels", func() {
+		var preset *v1.VirtualMachineInstancePreset
+		labelKey := "kubevirt.io/cpu"
+		labelValue := "dodecacore"
+		numCores := uint32(12)
+		presetName := "twelve-cores"
+
+		var annotationLabel string
+		var annotationVal string
+
+		var vmiWin7 *v1.VirtualMachineInstance
+		var vmiWin10 *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			selector := k8smetav1.LabelSelector{
+				MatchLabels: map[string]string{
+					labelKey: labelValue,
+				},
+			}
+
+			preset = &v1.VirtualMachineInstancePreset{
+				ObjectMeta: k8smetav1.ObjectMeta{Name: presetName},
+				Spec: v1.VirtualMachineInstancePresetSpec{
+					Selector: selector,
+					Domain: &v1.DomainSpec{
+						CPU: &v1.CPU{Cores: numCores},
+					},
+				},
+			}
+
+			// The actual type of machine is unimportant here. This test is about the label
+			vmiWin7 = tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+			vmiWin7.Labels = map[string]string{labelKey: labelValue}
+			vmiWin10 = tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+			vmiWin10.Labels = map[string]string{labelKey: labelValue}
+
+			annotationLabel = fmt.Sprintf("virtualmachinepreset.kubevirt.io/%s", presetName)
+			annotationVal = v1.GroupVersion.String()
+		})
+
+		It("[test_id:672] Should match multiple VMs via MatchLabel", func() {
+			By("Creating preset with MatchExpression")
+			err := virtClient.RestClient().Post().Resource("virtualmachineinstancepresets").Namespace(tests.NamespaceTestDefault).Body(preset).Do().Error()
+			Expect(err).ToNot(HaveOccurred())
+
+			// Give virt-api's cache time to sync before proceeding
+			time.Sleep(3 * time.Second)
+
+			By("Checking that preset was created")
```

Comment:
you don't need to check that the preset is created.

Reply:
Upon closer inspection, I agree this isn't in the test plan. Checking for error on creation is enough.